src/cli/utils/testdata/opts.go

```diff
@@ -497,8 +498,11 @@ func (MockBackupGetter) Backup(
 	return nil, errors.New("unexpected call to mock")
 }
 
-func (MockBackupGetter) Backups(context.Context, []model.StableID) ([]*backup.Backup, error) {
-	return nil, errors.New("unexpected call to mock")
+func (MockBackupGetter) Backups(
+	context.Context,
+	[]model.StableID,
+) ([]*backup.Backup, *fault.Errors) {
+	return nil, fault.New(false).Fail(errors.New("unexpected call to mock"))
 }
```

Comment:
The naming is interesting because it always returns an error.

Reply:
It's a bit of a standard pattern for mocks.  The mock has to comply with an interface; but if certain calls on the interface aren't expected, it returns an error to showcase that the mock isn't set up to handle that function.